workshop/content/DataBrew/01_Setting_up.en.md

```diff
@@ -8,28 +8,29 @@ pre = "<b>0.1. </b>"
 
 In this section we will setup an AWS Glue DataBrew project which is an interactive data preparation workspace in DataBrew. During this process we will create the following:
 
-- **Dataset:** rows or records that are divided into columns or fields.
```

Comment:
Used the definition from the [docs](https://docs.aws.amazon.com/databrew/latest/dg/datasets.html)